_overviews/scala3-migration/incompat-other-changes.md

```diff
@@ -71,22 +79,26 @@ The [Scala 3 migration compilation](tooling-migration-mode.html) can automatical
 
 The Scala 3 compiler requires the constructor of private classes to be private.
 
+{% tabs scala-2-constructor_1 %}
+{% tab 'Scala 2 Only' %}
 For instance, in the example:
```

Comment:
This is very minor, but we should put only the code snippet in the tab, not this sentence. The only situation where we put other things in a tab is when there are multiple tabs showing different pieces of code and the text need to refer specifically to the content of each snippet.

Reply:
Okay, sorry for the mistake, I'm correcting it on all the other PRs.